pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -162,6 +163,17 @@ func (r *ReconcileAccountClaim) Reconcile(request reconcile.Request) (reconcile.
 		return reconcile.Result{}, nil
 	}
 
+	isCCS := accountClaim.Spec.BYOCAWSAccountID != ""
+
+	if accountClaim.Status.State == awsv1alpha1.ClaimStatusPending {
+		now := metav1.Now()
+		unclaimedCondition := controllerutils.FindAccountClaimCondition(accountClaim.Status.Conditions, awsv1alpha1.AccountUnclaimed)
```

Comment:
This AccountClaimCondition part is not working for CCS accounts.  I checked my `make create-ccs` accountclaim and the only status condition it has at all is `type: Claimed`.  I think we could simplify this to something like:

```go
if accountClaim.Status.State == awsv1alpha1.ClaimStatusPending {
  now := metav1.Now()
  pendingDuration := now.Sub(accountClaim.ObjectMeta.CreationTimestamp)
  localmetrics.Collector.SetAccountClaimPendingDuration(isCCS, pendingDuration.Seconds())
}
```

 I'm not sure if that's the correct way to use it offhand, but lets use the creation timestamp.  The SLO is from Creation Time to Ready, so it's our best indicator, as the accountClaim is created when the customer wants a cluster for both CCS and non-CCS and is Ready when it's done either way as well.